ui/perfherder/alerts/AlertsView.jsx

```diff
@@ -55,43 +55,30 @@ class AlertsView extends React.Component {
   }
 
   componentDidUpdate(prevProps, prevState) {
-    const { count, frameworkOptions } = this.state;
-    const { validated } = this.props;
-    const prevValitated = prevProps.validated;
+    const { count } = this.state;
 
     if (prevState.count !== count) {
       this.setState({ totalPages: this.generatePages(count) });
     }
 
-    // filters updated directly in the url
```

Comment:
FYI @ionutgoldan The reason this block was removed is because this should not be necessary. One of the changes that is unavoidable with these routing changes is that manually changing query parameters will cause a full page reload.